### PR TITLE
Some elbow tracking info

### PR DESCRIPTION
**Pull request by Louka3000**
TODO:
- Component lists page
- Body proportion image (with purple person)

### CODE REVIEW

**Comment by Louka3000** [comment]
Can merge this PR now btw.

Current component lists page seems to only work with 5 full trackers, then it just adds more IMUs.
I know some Javascript but I'd prefer someone else with more experience to do it :p 
As for the body proportion image, I'd like @calliePepper to help make pictures :3

**Review by calliePepper — APPROVED** [review]
Amazing, now I gotta make a little picture :3

File: server-setup/body-config.md [hunk 1]
```diff
@@ -134,19 +134,19 @@ Make sure to follow this order (configure head first and feet last)
 ##### Head offset (8-12)
 {:.no_toc}
 
-Shake your head left to right as if you're disagreeing. All trackers should stay in place as you do so, any movement should be negligible. If they move too much, adjust the head offset.
+Shake your head left to right as if you’re disagreeing. Adjust your head offset until any movement is be negligible. All trackers should stay in place.
```
Comment (Eirenliel) [inline server-setup/body-config.md:137]:
```suggestion
Shake your head left to right as if you’re disagreeing. Adjust your head offset until any movement is negligible. All trackers should stay in place. 
```

File: server-setup/body-config.md [hunk 2]
```diff
@@ -134,19 +134,19 @@ Make sure to follow this order (configure head first and feet last)
...
 {:.no_toc}
 
-Shake your head left to right as if you're disagreeing. All trackers should stay in place as you do so, any movement should be negligible. If they move too much, adjust the head offset.
+Shake your head left to right as if you’re disagreeing. Adjust your head offset until any movement is be negligible. All trackers should stay in place. 
 
 ##### Neck Length (8-14)
 {:.no_toc}
 
-Move your head up and down as if you're nodding. All trackers should stay in place as you do so, any movement should be negligible. If they move too much, adjust your neck length.
+Move your head up and down as if you’re nodding OR move head side-to-side as if you're confused. Adjust your neck length until any movement is be negligible. All trackers should stay in place.
```
Comment (Eirenliel) [inline server-setup/body-config.md:142]:
```suggestion
Move your head up and down as if you’re nodding OR move head side-to-side as if you're confused. Adjust your neck length until any movement is negligible. All trackers should stay in place.
```

**Review by Eirenliel — CHANGES_REQUESTED** [review]

**Review by Eirenliel — APPROVED** [review]

**Pull request merged**